yahoo/libymsg.c

```diff
@@ -4923,8 +4923,9 @@ unsigned int yahoo_send_typing(PurpleConnection *gc, const char *who, PurpleTypi
 		yahoo_packet_hash(pkt, "ssssss", 49, "TYPING", 1, purple_connection_get_display_name(gc),
                   14, " ", 13, state == PURPLE_TYPING ? "1" : "0",
                   5, fed_who, 1002, "1");
-        if (fed)
+        if (fed) {
         	yahoo_packet_hash_int(pkt, 241, fed);
+        }
```

Comment:
AFAICT, the whole file uses tabs, so might as well fix that while here.
```suggestion
		if (fed) {
			yahoo_packet_hash_int(pkt, 241, fed);
		}
```